hack/make/ubuntu

```diff
@@ -150,6 +158,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
Yes, that's exactly what I mean.  In a properly-configured system,
dependencies from Recommends get installed automatically.  On Debian
systems, AppArmor is not installed by default, but the very act of
installing Docker will bring it in, which is a very disruptive change.  I
think this is properly solved by deprecating this "hack/make/ubuntu"
package entirely in favor of our new packages where this dependency is
already properly added to Recommends for Ubuntu-based platforms.